pom.xml

```diff
@@ -35,46 +36,54 @@
   </licenses>
 
   <dependencies>
+
     <dependency>
-      <groupId>io.jenkins-x.client</groupId>
-      <artifactId>jx-java-client</artifactId>
-      <version>1.0.4</version>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>git</artifactId>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-job</artifactId>
-      <version>2.15</version>
+      <artifactId>workflow-api</artifactId>
     </dependency>
 
     <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>git</artifactId>
-      <version>3.6.4</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.jenkins-ci</groupId>
-          <artifactId>annotation-indexer</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-job</artifactId>
     </dependency>
 
+
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-api</artifactId>
-      <version>2.22</version>
+      <artifactId>workflow-cps</artifactId>
     </dependency>
 
+
+    <!-- non bom dependencies -->
     <dependency>
       <groupId>org.jenkins-ci.plugins.pipeline-stage-view</groupId>
       <artifactId>pipeline-rest-api</artifactId>
-      <version>2.9</version>
+      <version>2.13</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.jenkins-x.client</groupId>
+      <artifactId>jx-java-client</artifactId>
+      <version>1.0.4</version>
+    </dependency>
+
+    <!-- test dependencies -->
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-cps</artifactId>
-      <version>2.41</version>
+      <artifactId>workflow-aggregator</artifactId>
```

Comment:
You are urged _not_ to use `workflow-aggregator` as a `test` dependency. Instead, add dependencies on particular plugins you need (transitive dependencies will be pulled in as needed): always `workflow-job` + `workflow-cps`, typically also `workflow-basic-steps` and/or `workflow-durable-task-step`. With the BOM this becomes much easier.